src/lib.rs

```diff
@@ -155,6 +155,23 @@ impl Indent {
         }
         indent
     }
+
+    pub fn to_string_no_align(&self, config: &Config) -> String {
```

Comment:
The behaviour is subtly different. It does not add on the extra alignment to the "if" statement, only the block indent. Unless you believe it's preferable to clone() the config passed into rewrite_if_else and to modify it to have an alignment of 0.

I did this to resolve an issue with Allman and Stroupstrup styles having bizarrely aligned braces for "let x = if ..." statements, where the opening brace would line up with the if, but the closing brace would line up with the "let".